documentation/cluster-controller.adoc

```diff
@@ -224,6 +227,34 @@ configuration file. For this reason, you can find more information on how to use
 
 For more information on how metrics work, the related documentation is available link:../../metrics/METRICS.md[here]
 
+===== Topic controller
+
+Alongside the Kafka cluster and the Zookeeper ensemble, the cluster controller can deploy the topic controller as well.
+In order to do that, the 'topic-controller-config` field has to be put into the data section of the cluster ConfigMap
+containing a JSON string with the topic controller configuration.
+Without this field, the cluster controller doesn't deploy the topic controller and the user has to do that using the
```

Comment:
"... doesn't deploy the topic controller. It is still possible to deploy the topic controller by creating appropriate Kubernates/OpenShift resources."